### PR TITLE
chore: modify lua syntax error

### PR DESCRIPTION
**Pull request by zanehy**
Modify luacheck exception code and add luacheck configuration file.

### CODE REVIEW

File: apioak/admin/project.lua [hunk 1]
```diff
@@ -2,6 +2,7 @@ local db         = require("apioak.db")
 local pdk        = require("apioak.pdk")
 local schema     = require("apioak.schema")
 local controller = require("apioak.admin.controller")
+local _
```
Comment (shuaijinchao) [inline apioak/admin/project.lua]: Why is that?

Comment (zanehy) [reply apioak/admin/project.lua]: Because '_' is used many times in the code to make a unified definition。

Comment (shuaijinchao) [reply apioak/admin/project.lua]: `_` this is an ignored variable placeholder, normally it does not require explicit declaration.

http://lua-users.org/wiki/LuaStyleGuide

https://github.com/mpeterv/luacheck/issues/40

Comment (zanehy) [reply apioak/admin/project.lua]: OK, thanks.

Comment (shuaijinchao) [reply apioak/admin/project.lua]: LGTM, cool

**Pull request merged**